website/static/js/fangorn.js

```diff
@@ -737,68 +843,100 @@ function _fangornUploadMethod(item) {
  * @returns {Array} Returns an array of mithril template objects using m()
  * @private
  */
-function _fangornActionColumn (item, col) {
-    var self = this,
+function _fangornDefineToolbar (item) {
```

Comment:
The docstring for this function is no longer accurate; it seems to be defining an array of button objects and assigning it to `item.icons`.